crates/miden-lib/asm/kernels/transaction/api.masm

```diff
@@ -326,98 +330,98 @@ end
 
 #! Gets an item from the account storage.
 #!
-#! Inputs:  [index, pad(15)]
+#! Inputs:  [name_id_prefix, name_id_suffix, pad(14)]
 #! Outputs: [VALUE, pad(12)]
 #!
 #! Where:
-#! - index is the index of the item to get.
+#! - name_id_{prefix, suffix} are the prefix and suffix felts of the name identifier, which are
+#!   the first two felts of the hashed slot name.
 #! - VALUE is the value of the item.
 #!
 #! Panics if:
-#! - the index is out of bounds.
+#! - a slot with the provided name ID does not exist in account storage.
 #!
 #! Invocation: dynexec
 export.account_get_item
     # authenticate that the procedure invocation originates from the account context
     exec.authenticate_account_origin
-    # => [storage_offset, storage_size, index, pad(15)]
-
-    # apply offset to storage slot index
-    exec.account::apply_storage_offset
-    # => [index_with_offset, pad(15)]
+    # => [name_id_prefix, name_id_suffix, pad(14)]
 
     # fetch the account storage item
     exec.account::get_item
-    # => [VALUE, pad(15)]
+    # => [VALUE, pad(14)]
 
     # truncate the stack
-    movup.4 drop movup.4 drop movup.4 drop
+    movup.4 drop movup.4 drop
     # => [VALUE, pad(12)]
+
+    # Drop extraneous elements that get_item pushed onto the stack.
+    movup.4 drop
```

Comment:
This should probably be combined with line 355 above. But also, this is inconsistent with the stack state on line 352 - i.e., either it should be `pad(15)`, or we shouldn't need this `drop`.